app/views/marine/index.html.erb

```diff
@@ -2,7 +2,7 @@
 <%= content_for :social_title, "Explore the World's Marine Protected Areas" %>
 <%= content_for :social_description, "Over 70% of the surface of Earth is ocean, comprised of highly diverse ecosystems, and providing a wide range of marine ecosystem services that support human society, health and the economy. This website presents the most recent official coverage statistics for marine protected areas, updated monthly from the World Database on Protected Areas." %>
 <%= content_for :social_image, image_url('social-marine.png') %>
-<%= content_for :social_image_alt, "A map of the world with highlighted areas that represent protected areas. The following statistics are shown: there are 15,271 marine protected areas, #{@oceanProtectedAreasPercent} of the ocean is covered by protected areas and the total area protected is 23,036,078km² These statistics are for 2017." %>
+<%= content_for :social_image_alt, "A map of the world with highlighted areas that represent protected areas. The following statistics are shown: there are #{@marine_statistics["total_marine_protected_areas"]} marine protected areas, #{@marine_statistics["total_ocean_pa_coverage_percentage"]} of the ocean is covered by protected areas and the total area protected is 23,036,078km² These statistics are for 2017." %>
```

Comment:
Can we please use single quotes here rather then double quotes while getting the value from the hash? So it's less confusing as the entire content is surrounded by double quotes already.

Reply:
Sure I will update that now @Levia